templates/download/xilinx/index.html

```diff
@@ -32,77 +32,298 @@ <h1>Install Ubuntu on Xilinx</h1>
 </section>
 
 <section class="p-strip--light">
```

Comment:
I think this should be a white strip 
```suggestion
<section class="p-strip">
```